app/services/auctions_closed_within_last_twenty_four_hours_finder.rb

```diff
@@ -0,0 +1,9 @@
+class AuctionsClosedWithinLastTwentyFourHoursFinder
```

Comment:
I'm ok with that ... I was also wondering why we instead don't create and use methods on `AuctionQuery` for this.